src/index.ts

```diff
@@ -88,20 +98,23 @@ window.Spotfire.initialize(async (mod) => {
         //for testing purposes
         //renderParetoAsTextInConsole(pareto, {} as Settings);
         context.signalRenderComplete();
+        mod.controls.errorOverlay.hide();
     }
 });
 
 /**
- * Validate that no empty path element is followed by a value and that all values are positive.
- * @param rootNode - The hierarchy root.
+ * Validate that all values are positive
+ * @param pareto
+ * @returns null if all values are positive, message if there are any negative values
  */
-function validateDataView(rootNode: DataViewHierarchyNode): string[] {
-    let warnings: string[] = [];
-    let rows = rootNode.rows();
+function validateDataView(pareto: Pareto): string | null {
+    let warning: string | null = null;
 
-    //to do: validate data, check if there are negative values, or values outside some range, etc
+    if (pareto.minValue < 0) {
+        warning = "The pareto chart can't contain any negative values";

```

Comment:
Pareto with capital P. Perhaps we can change this to: "A Pareto chart can not contain any negative values"?